content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -242,6 +242,13 @@ where `config.yaml` contains the custom `imageRepository`, and/or `imageTag`
 for etcd and CoreDNS.
 * Pass the same `config.yaml` to `kubeadm init`.
 
+#### Custom sandbox (pause) images
+
+Container runtimes are responsible for their sandbox images, a common default being `k8s.gcr.io/pause`.
+To set a custom image for these you need to [configure the container-runtime](/docs/setup/production-environment/container-runtimes/#overriding-the-sandbox-pause-image)
```

Comment:
Here we can just link to the main CR page ALA "find how to setup the pause image for your CR of choice"